refarch-integrations/refarch-address-integration/refarch-address-integration-client/src/main/java/de/muenchen/refarch/integration/address/client/impl/StreetsMunichImpl.java

```diff
@@ -0,0 +1,37 @@
+package de.muenchen.refarch.integration.address.client.impl;
+
+import de.muenchen.refarch.integration.address.client.api.StreetsMunichApi;
+import de.muenchen.refarch.integration.address.client.exception.AddressServiceIntegrationClientErrorException;
+import de.muenchen.refarch.integration.address.client.exception.AddressServiceIntegrationException;
+import de.muenchen.refarch.integration.address.client.exception.AddressServiceIntegrationServerErrorException;
+import de.muenchen.refarch.integration.address.client.model.request.ListStreetsModel;
+import de.muenchen.refarch.integration.address.client.gen.api.StraenMnchenApi;
+import de.muenchen.refarch.integration.address.client.gen.model.Strasse;
+import de.muenchen.refarch.integration.address.client.gen.model.StrasseResponse;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@RequiredArgsConstructor
+@Slf4j
+public class StreetsMunichImpl implements StreetsMunichApi {
+
+    private final StraenMnchenApi straessenMuenchenApi;
```

Comment:
Why were the `s` cut out from the ClassName `StraenMnchenApi` instead of `StrassenMuenchenApi`? Is this a code generation problem due to the provided API-Spec?

Reply:
Yes